src/libaktualizr/package_manager/fetcher_test.cc

```diff
@@ -249,11 +249,17 @@ int main(int argc, char** argv) {
                                               std::string("--create"));
   TemporaryDirectory temp_dir;
   // Utils::copyDir doesn't work here. Complaints about non existent symlink path
-  int r = system((std::string("cp -r ") + argv[1] + std::string(" ") + temp_dir.PathString()).c_str());
```

Comment:
Does this mean we are no longer using `argv[1]`? If so, we should stop passing it in.